packages/ngx-flicking/projects/ngx-flicking/package.json

```diff
@@ -1,12 +1,28 @@
 {
-  "name": "ngx-flicking",
-  "version": "0.1.0",
+  "name": "@egjs/ngx-flicking",
+  "version": "3.0.0-beta",
+  "description": "A module used to implement flicking interactions. With this module, you can make flicking gestures, which are ways to navigate left and right to move between panels arranged side by side.",
   "peerDependencies": {
     "@angular/common": "^7.2.0",
-    "@angular/core": "^7.2.0"
+    "@angular/core": "^7.2.15"
```

Comment:
Which Angular feature made this change?

Reply:
@WoodNeck It is not intended change. But it would not be problem by following reason.
  1. It's patch (bug fix)
  2. It's peerDependency